style.css

```diff
@@ -73,3 +78,202 @@ header {
   height: 20px;
   width: 20px;
 }
+
+.works {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: 355px;
+  grid-template-rows: auto;
+  row-gap: 100px;
+  margin-bottom: 150px;
+}
+
+.grid-row {
+  /* margin-top: 150px; */
+  width: 355px;
+  border-radius: 15px;
+  border: 1px rgba(128, 128, 128, 0.286) solid;
+}
+
+.card {
+  display: flex;
+  justify-content: center;
+}
+
+.image-container {
+  margin-top: 20px;
+  width: 315px;
+  height: 235px;
+}
+
+.image-container img {
+  width: 315px;
+  height: 235px;
+}
+
+.project {
+  display: flex;
+  justify-content: center;
+}
+
+.project-name {
+  width: 315px;
+  font-size: 35px;
+  color: #172b4d;
+  font-weight: bold;
+  padding-top: 10px;
+}
+
+.specs {
+  display: flex;
+  justify-content: center;
+  padding-top: 15px;
+}
+
+.project-specs {
+  display: flex;
+  width: 315px;
+  font-size: 14px;
+  font-weight: 500;
+  color: #344563;
+}
+
+.counter {
+  padding: 0 10px;
+}
+
+.description {
+  display: flex;
+  justify-content: center;
+  padding-top: 20px;
+}
+
+.project-description {
+  width: 315px;
+  color: #344563;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+}
+
+.tools {
+  display: flex;
+  justify-content: center;
+  padding-top: 25px;
+}
+
+.tools-container {
+  width: 315px;
+}
+
+.project-tools {
+  display: flex;
+  justify-content: start;
+}
+
+.tool {
+  display: inline-block;
+  background-color: #ebebff;
+  padding: 4px 12px;
+  border-radius: 8px;
+  margin-right: 7px;
+  color: #6070ff;
+  font-size: 15px;
+}
+
+.card-button {
+  display: flex;
+  justify-content: center;
+  padding: 30px 0;
+}
+
+.card-button-style {
+  width: 315px;
+}
+
+button {
+  padding: 10px;
+  background-color: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-size: 25px;
+  color: #396df2;
+  border: 1px solid #396df2;
+  border-radius: 10px;
+}
+
```

Comment:
- Hey, please, let's reduce the font size to 17px to match the design as much as we can from Figma
- Let's also add the interacting effect of the button `hover, disabled, active` states to improve the user experience when interacting with the button below:

*figma*
![Screenshot from 2023-01-14 01-35-32](https://user-images.githubusercontent.com/59397159/212441779-22958641-a6b1-4d5d-a2fb-8a7c694f49f5.png)

*your design*
![Screenshot from 2023-01-14 01-35-21](https://user-images.githubusercontent.com/59397159/212441781-f82b63f3-ce95-4777-b79f-acca8c6a9077.png)

---

*button actions*


![Screenshot from 2023-01-14 01-40-31](https://user-images.githubusercontent.com/59397159/212441877-d880c9d2-99e7-40fc-b64b-d920ec400a7e.png)